themes/bootstrap3/less/components/sidebar.less

```diff
@@ -104,6 +104,26 @@
   overflow: hidden;
 }
 
+.multiFacetSelection {
+  .facet {
+    a.exclude {
+      outline: none;
```

Comment:
Are we [accounting for hiding the keyboard focus styles](https://www.outlinenone.com/) elsewhere?